app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::ReviewsController < ApplicationController
+  def index
+    item = Item.find(params[:item_id])
+    reviews = item.reviews
+    json_response(reviews)
```

Comment:
I would follow the established convention of the app and make sure to be explicit about both the structure and what fields are being exposed from the API.